src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -21,26 +21,25 @@
 
 (def row-height-px 56)
 
-(defn- disabled-text [protected?]
-  (if protected?
-    (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
-         "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account.")
-    "This workspace is protected. To access the data, please contact help@firecloud.org."
-    ))
+(def dbGap-disabled-text
+  (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
+       "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account."))
```

Comment:
Same answer as before... it's displayed "in an anchor title which unfortunately preserves the whitespace (including embedded newline) when rendering".